app/templates/main/browse.html

```diff
@@ -0,0 +1,270 @@
+{%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
+
+{% block pageTitle %}Browse – {{config['SERVICE_NAME']}} – GOV.UK{% endblock %}
+
+
+{% block content %}
+<div class="govuk-grid-row browse__page">
+  {{ super() }}
+
+  <div class="search__container govuk-grid-column-full">
+    <div class="search__container__content">
+      <p class="govuk-body search__heading">Search for digital records</p>
+      <form method="POST">
+        {{ form.csrf_token }}
+        <div class="govuk-form-group govuk-form-group__search-form">
+          <label for="searchInput"></label>
+          <input class="govuk-input govuk-!-width-three-quarters" id="searchInput" name="query" type="text">
+          <button class="govuk-button govuk-button__search-button" data-module="govuk-button" type="submit">Search</button>
+        </div>
+        <p class="govuk-body-s">Search using a record metadata term, for example – transferring body, series,
+          consignment
+          ref etc.</p>
+      </form>
+    </div>
+  </div>
+  {% if results %}
+  <h1 class="govuk-heading-l browse__records-found__text">Records found {{ num_records_found }}</h1>
+  <p class="govuk-body browse__body">You are viewing</p>
+  <p class="govuk-body-m browse__body browse__available__text">Everything available to you</p>
+  <div class="govuk-form-group sort-container__form">
+    <div class="browse__sort-container">
+      <label class="govuk-label" for="sort">
+        Sort by
+      </label>
+      <select class="govuk-select govuk-select__sort-container-select" id="sort" name="sort">
+        <option value="body-a">Transferring body (A to Z)</option>
+        <option value="body-b" selected>Transferring body (Z to A)</option>
+        <option value="series-a">Series (A to Z)</option>
+        <option value="series-b">Series (Z to A)</option>
+        <option value="date-first">Date record transferred (most recent first)</option>
+        <option value="date-last">Date record transferred (oldest first)</option>
+      </select>
+      <button class="govuk-button govuk-button__sort-container-update-button" id="sort-update-button" data-module="govuk-button">
+        Apply
+      </button>
+    </div>
+  </div>
+
+  <div class="govuk-width-container">
+
+    <!-- FILTERS -->
+    <div class="govuk-grid-column-one-third filters-form">
+
+      <div class="browse-filter-container">
+        <div class="browse-filter__header">
+          <p class="govuk-body-l browse__body filters-form__title">Filter within browse</p>
+          <img src="{{ url_for('static', filename='image/filter-icon.svg') }}" class="browse-filter__icon"
+            alt="filter-icon">
+        </div>
+        <p class="govuk-body browse__body">Transferring body</p>
+        <div class="govuk-form-group filters-form__group filters-form__transferring-body-group">
+          <select class="govuk-select govuk-select__filters-form-transferring-body-select" id="sort" name="sort">
```

Comment:
Do these filters do anything without JavaScript enabled? They don't appear to be in a `<form>` element.

Reply:
The filters will be calculated on the backend, we don't have any JS features yet so will be sure to put this inside of a form.